src/siteSettings/components/SiteCheckoutSettings/SiteCheckoutSettings.tsx

```diff
@@ -0,0 +1,83 @@
+import { Card, CardContent, TextField, Typography } from "@material-ui/core";
+import CardTitle from "@saleor/components/CardTitle";
+import FormSpacer from "@saleor/components/FormSpacer";
+import { ShopErrorFragment } from "@saleor/fragments/types/ShopErrorFragment";
+import { getFormErrors } from "@saleor/utils/errors";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import { SiteSettingsPageFormData } from "../SiteSettingsPage";
+import { messages } from "./messages";
+
+interface SiteCheckoutSettingsProps {
+  data: SiteSettingsPageFormData;
+  errors: ShopErrorFragment[];
+  disabled: boolean;
+  onChange: (event: React.ChangeEvent<any>) => void;
+}
+
+const SiteCheckoutSettings: React.FC<SiteCheckoutSettingsProps> = ({
```

Comment:
Maybe we could call `SiteCheckoutSettingsCard` so its mor explicit

Reply:
+1

Reply:
well, might be, ok